wagtail/admin/views/pages/edit.py

```diff
@@ -79,15 +81,46 @@ def send_commenting_notifications(self):
         if not new_comments and not deleted_comments and not resolved_comments:
             return
 
-        # Get subscribers
+        relevant_comment_ids = [comment.pk for comment in new_comments + resolved_comments]
```

Comment:
Not sure why we need to add new_comments here since this is list is only used for handling emails about individual threads?

Reply:
Good point, we don't! Will remove